lib/sablon/template.rb

```diff
@@ -32,6 +33,10 @@ def render(context, properties = {})
             out.write(process(Processor::Document, content, env))
           elsif entry_name == 'word/numbering.xml'
             out.write(process(Processor::Numbering, content))
+          elsif entry_name == 'word/_rels/document.xml.rels'
+            out.write(process(Processor::Image, content, properties))
```

Comment:
A lot of different stuff gets put in the *.rels files and there can be several of them: header#, footer#, document, etc. Which one gets used depends on the xml document it pertains to, i.e. I have an image in header2.xml, the relationship gets stuck in header2.xml.rels. 

I don't know if this is the best way to handle it given the nature of *.rels files. Since to work with the general case you need to consider the primary XML document and then open the corresponding rels file. In my mind you would open and edit the corresponding *.rels file on an as needed basis to append information. Instead of calling `Processor:Image` in the `Zip::File` block your image processor class would know which document you are working in, be it the main doc, header or footer and then pass that information onto the proper rels file. 

However, that scheme doesn't play well with the current structure of sablon since it loops over the files only once. I think you would need a separate `Processor::Rels` class instance that gets registered to the current Environment instance to handle passing information onto a separate *.rels file. Kind of how the Numbering class gets called as needed to pass information to numbering.xml. Maybe I am over thinking things though?